CHANGELOG.rst

```diff
@@ -25,6 +25,12 @@ Fixed
 
   Reported by John Arnold.
 
+* Fix a bug where sensor watch queues were not deleted after sensor container process was shut
```

Comment:
Yeah, I documented that script in the issue already. I just didn't know where to place that documentation. st2docs seemed too much. Ideas welcome.